### PR TITLE
Beta2 trailblazer test2

### PR DESCRIPTION
**Pull request by KINGBEAKSUU**

### CODE REVIEW

**Comment by rav4kumar** [comment]
thanks for the clean pr! Will get it merged!

next time please pr to https://github.com/dragonpilot-community/dp-devel

Cheers,

**Comment by Stupefacient** [comment]
Also, don't include personal modifications such as your driver monitoring changes next time. Only things relevant to the actual pull request

Thanks

File: selfdrive/car/gm/interface.py [hunk 1]
```diff
@@ -227,6 +239,14 @@ def _update(self, c):
 
     # Enabling at a standstill with brake is allowed
     # TODO: verify 17 Volt can enable for the first time at a stop and allow for all GMs
+    #below_min_enable_speed = ret.vEgo < self.CP.minEnableSpeed or self.CS.moving_backward
```
Comment (rav4kumar) [inline selfdrive/car/gm/interface.py:242]: wont be adding this. this look like personal addons.

File: selfdrive/car/gm/carstate.py [hunk 2]
```diff
@@ -44,6 +44,7 @@ def update(self, pt_cp, cam_cp, loopback_cp):
     ret.vEgoRaw = mean([ret.wheelSpeeds.fl, ret.wheelSpeeds.fr, ret.wheelSpeeds.rl, ret.wheelSpeeds.rr])
     ret.vEgo, ret.aEgo = self.update_speed_kf(ret.vEgoRaw)
     # sample rear wheel speeds, standstill=True if ECM allows engagement with brake
+    ret.vEgoCluster = ret.vEgo * 1.0675
```
Comment (rav4kumar) [inline selfdrive/car/gm/carstate.py:47]: personal addons

File: selfdrive/monitoring/driver_monitor.py [hunk 3]
```diff
@@ -18,10 +18,10 @@ class DRIVER_MONITOR_SETTINGS():
   def __init__(self, DT_DMON=DT_DMON):
     self._DT_DMON = DT_DMON
     # ref (page15-16): https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:42018X1947&rid=2
-    self._AWARENESS_TIME = 30. # passive wheeltouch total timeout
-    self._AWARENESS_PRE_TIME_TILL_TERMINAL = 15.
-    self._AWARENESS_PROMPT_TIME_TILL_TERMINAL = 6.
-    self._DISTRACTED_TIME = 11. # active monitoring total timeout
+    self._AWARENESS_TIME = 99999. # passive wheeltouch total timeout
```
Comment (rav4kumar) [inline selfdrive/monitoring/driver_monitor.py:21]: against commas safety. cannot merge this

Comment (KINGBEAKSUU) [reply selfdrive/monitoring/driver_monitor.py:21]: only for test i will change this

File: selfdrive/car/gm/interface.py (hunk 1 above)
Comment (KINGBEAKSUU) [reply selfdrive/car/gm/interface.py:242]: ok

File: selfdrive/car/gm/carstate.py (hunk 2 above)
Comment (KINGBEAKSUU) [reply selfdrive/car/gm/carstate.py:47]: ok

**Pull request closed without merging**